client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -27,11 +27,109 @@
         style="max-height: 52vh;"
       />
     </b-form-group>
+
+    <hr>
+
+    <div>
+      <b-form-group
+        label-size="lg"
+      >
+        <template #label>
+          <div
+            class="d-flex"
+          >
+            {{ $t('record.fieldConditions.label') }}
+
+            <b-button
+              variant="link"
+              class="p-0 ml-1"
+              :disabled="block.options.fields.filter(f => !f.isRequired).length === block.options.fieldConditions.length"
```

Comment:
text-decoration-none, otherwise people think its a link, if you remember we talked about this